tools/psa/release.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# Copyright (c) 2017-2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import sys
+import shutil
+
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+sys.path.insert(0, ROOT)
+from tools.targets import Target, TARGET_MAP, TARGET_NAMES
+
+PYTHON_EXEC = 'python' if os.path.basename(sys.executable).startswith('mbed') else sys.executable
```

Comment:
I'm not sure why you would expect that `mbed` would be in the executable name. What caused you to write this code?

Reply:
Mbed CLI has this code so that it can be run as a py2exe or py2app thing. You don't need it.